maintainers/maintainer-list.nix

```diff
@@ -2905,6 +2905,16 @@
     githubId = 11909469;
     name = "Fabian Geiselhart";
   };
+  fab = {
+    email = "mail@fabian-affolter.ch";
+    name = "Fabian Affolter";
+    github = "fabaff";
+    githubId = 116184;
+    keys = [{
+      longkeyid = "dsa1024/0xE23CD2DD36A4397F";
```

Comment:
:scream:

Reply:
It is. `dsa1024` was the default in 2008 or so as far as I remember.

If you prefer then I can change it to one that using different settings .

Reply:
Yeah, I get it. Totally up to you.